Description: add pyproject.toml to support PEP-517-compatible build-systems
 .
 This makes it possible to build and install the module without directly
 invoking setup.py which has been deprecated.
 .
 Retain the setup.py script for backwards-compatibility.
Last-Update: 2023-08-03
Author: Jeremy Sowden <jeremy@azazel.net>
Origin: upstream, http://git.netfilter.org/nftables/commit/?id=8e603e0f7eec7c0000344a004228a30fbf0ece5c
---
 py/Makefile.am    | 2 +-
 py/pyproject.toml | 3 +++
 3 files changed, 6 insertions(+), 2 deletions(-)
 create mode 100644 py/pyproject.toml

--- a/py/Makefile.am
+++ b/py/Makefile.am
@@ -1 +1 @@
-EXTRA_DIST = setup.cfg setup.py src
+EXTRA_DIST = pyproject.toml setup.cfg setup.py src
--- /dev/null
+++ b/py/pyproject.toml
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
